source/scripts/ev3dev-powerinfo

```diff
@@ -0,0 +1,41 @@
+#!/usr/bin/env bash
+
+
+print_val() {
+  printf "%-20s%s\n" "$1: " "$2"
+}
+
+print_power_info() {
+    for pwr_supply in /sys/class/power_supply/*; do
+        uVolts=`cat $pwr_supply/voltage_now`
+        intVolts=`expr $uVolts / 1000000`
+        decmicroVolts=`expr $uVolts - $intVolts \* 1000000`
+        decmiliVolts=`expr $decmicroVolts / 1000`
+        printf "%-20s: %3d.%03d  V\n" "Voltage" $intVolts $decmiliVolts
+
+
+        uAmps=`cat $pwr_supply/current_now`
```

Comment:
Only EV3 can measure current. This will cause an error on other devices.

Reply:
Oops. I didn't know that.

Reply:
Given this limitation, is it better that I just close the pull request?

Reply:
It is easy enough to wrap this section in:

    if [ -e "$pwr_supply/current_now" ]; then
        ...
    fi